### DISCUSSION TITLE
Submit issue directly from codespace

### DISCUSSION THREAD

**Discussion by nick-youngblut** [community/community#43319] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I changed my machine type from the 16-core option to the 2-core option, and now I cannot start the codespace. If I restart the codespace, I just get a message after a couple of minutes stating "codespace stopped", and I just have the option of "restart codespace". It would be helpful to have the option of directly submitting this issue (& logs) in addition to the only current option: "restart codespace".

**Comment by erikaellr** [comment]
Hi @nick-youngblut, we're thrilled you're using Codespaces, but sorry to hear you ran into this friction. We're always working to make GitHub and Codespaces as awesome as possible, and I agree, I think a feature like that would be really helpful. We can't guarantee that every bit of feedback sees action, but we are listening.

One thing you can try yourself without having to open a ticket on our [Support page](https://support.github.com/contact), is you could [export your changes to a branch](https://docs.github.com/en/codespaces/troubleshooting/exporting-changes-to-a-branch) from the codespace that's having the problem, so that you can safely delete it and resume work on a newly-created codespace. Hope this helps!

**Comment by nick-youngblut** [comment]
Thanks for the feedback! Maybe it would be helpful to include links on what to do when the codespace startup keeps resulting in errors? In my case, starting up the codespace via my web browser just keep sending me back to the "restart codespace" page, with no options on viewing logs or submitting a ticket on the support page.
